doc/en/weechat_faq.en.asciidoc

```diff
@@ -63,7 +63,8 @@ brew info weechat
 You can install WeeChat with this command:
 
 ----
-brew install weechat --with-python --with-perl
+brew install aspell
+brew install weechat --with-aspell --with-curl --with-guile --with-lua --with-perl --with-python --with-ruby
```

Comment:
From linuxbrew (I can check OS X when someone boots it, but both brews should be the same):

```
% brew info weechat
weechat: stable 1.1, HEAD
https://www.weechat.org
Not installed
From: https://github.com//homebrew/blob/master/Library/Formula/weechat.rb
==> Dependencies
Build: cmake ✘
Required: gnutls ✘, libgcrypt ✘, gettext ✘
Optional: guile ✘, aspell ✘, lua ✘, curl ✘
==> Options
--with-aspell
        Build with aspell support
--with-curl
        Build with brewed curl
--with-guile
        Build with guile support
--with-lua
        Build with lua support
--with-perl
        Build the perl module
--with-python
        Build with python support
--with-ruby
        Build the ruby module
--HEAD
        Install HEAD version
==> Caveats
Weechat can depend on Aspell if you choose the --with-aspell option, but
Aspell should be installed manually before installing Weechat so that
you can choose the dictionaries you want.  If Aspell was installed
automatically as part of weechat, there won't be any dictionaries.
```